go/api/standalone_client.go

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+package api
+
+// #cgo LDFLAGS: -L../target/release -lglide_rs
+// #include "../lib.h"
+import "C"
+
+// RedisClient is a client used for connection to standalone Redis servers.
+type RedisClient struct {
+	*baseClient
+}
+
+// NewRedisClient creates a [RedisClient] in standalone mode using the given [RedisClientConfiguration].
+func NewRedisClient(config *RedisClientConfiguration) (*RedisClient, error) {
```

Comment:
Please rename to `CreateClient` to match API of other clients

Reply:
Changed it to NewGlideClient and NewGlideClusterClient as there is no class and object concept in Go and this seems cleaner.